pkg/restore/import.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/pingcap/kvproto/pkg/kvrpcpb"
 	"github.com/pingcap/log"
 	restore_util "github.com/pingcap/tidb-tools/pkg/restore-util"
+	"github.com/pingcap/tidb/util/codec"
```

Comment:
CI failed, seems should use pd [codec](https://github.com/pingcap/br/pull/137/files#diff-eae46bd20abe35cad68c49bd179a3728R14)

Reply:
but the codec in pd and tidb seems no different

Reply:
Turns out I exchange `DecodeBytes` arguments by mistake.